_src/pattern-library/sass/components/_forms.scss

```diff
@@ -0,0 +1,484 @@
+
+    // section title or legend
+    .form-group-hd {
+        margin-bottom: spacing-vertical(small);
+        font-size: font-size(mid-large);
+    }
+
+    .field {
+        margin-bottom: spacing-vertical(base);
+
+        .field-hint {
+            font-size: font-size(small);
+            color: $field-tip-color;
+        }
+    }
+}
+
+// radio button and checkbox fieldsets
+.field-group {
+    margin-bottom: spacing-vertical(small);
+
+    // group title or legend
+    .field-group-hd {
+        margin-bottom: spacing-vertical(small);
+        font-size: font-size(mid-large);
+    }
+
+    .field {
+        margin-bottom: spacing-vertical(x-small);
+
+        .field-hint {
+            @include margin-left(spacing-horizontal(mid-large));
+            margin-top: spacing-vertical(xx-small);
+            font-size: font-size(small);
+            color: $field-tip-color;
+        }
+    }
+}
+
+.field-label {
+    display: block;
+    width: auto;
+    margin-bottom: spacing-vertical(x-small);
+    font-size: font-size(base);
+    line-height: 100%;
+    color: $label-color;
+
+    // presents the label inline with the form control
+    &.label-inline {
+        display: inline-block;
+        margin-bottom: 0;
+    }
+
+    // STATE: is selected
+    .field-input:checked + .field-input-label,
+    .field-radio:checked + .field-input-label,
+    &.is-active,
+    &.is-selected {
+        @include transition(all timing(xx-fast) ease-in-out 0s);
+        color: $label-color-active;
+    }
+}
+
+.field-hint {
+    margin-top: spacing-vertical(x-small);
+}
+
+.field-required {
+
+    @include ltr {
+
+        // for LTR displays we show the required indicator to the right of the label
+        &:after {
+            display: inline-block;
+            @include margin-left(spacing-horizontal(x-small));
+            content: "*";
+            font-size: font-size(small);
+            color: $field-required-indicator-color;
+        }
+    }
+
+    @include rtl {
+
+        // for RTL displays we show the required indicator to the left of the label
+        &:before {
+            display: inline-block;
+            @include margin-left(spacing-horizontal(x-small));
+            content: "*";
+            font-size: font-size(small);
+            color: $field-required-indicator-color;
+        }
+    }
+}
+
+.field-input,
+.field-select,
+.field-textarea {
+    @include placeholder { color: $input-placeholder-text; }
+    @include transition(all timing(xx-fast) ease-in-out 0s);
+    display: inline-block;
+    padding: rem($baseline/2);
+    border: 1px solid $input-border-default;
+    background: $input-background-default;
+    font-family: $font-family-sans-serif;
+    font-size: font-size(base);
+    color: $input-text-default;
+
+    // STATE: is active or has focus
+    &:focus,
+    &.is-active {
+        border-color: $input-border-default-focus;
+        color: $input-text-default-hover;
+        outline: none;
```

Comment:
It still receives an outline, just not the blue glow. I'm okay keeping the blue glow (I prefer it), but then we can't see the border colors you designed. The outline is meant to indicate where focus is. As long as we do that somehow, we're good. And I'm only removing the outline on the form fields - the rest of a web page would still use it. Happy to replace it though. Thoughts? cc @cptvitamin